### PR TITLE
Fixed Prefix on Network Topology

### PR DESCRIPTION
**Pull request by 0xAnon0602**
Added feature to automatic shift IPv version based on input 

https://github.com/user-attachments/assets/6bba2519-d104-4317-af52-ac816428d7b1

### CODE REVIEW

**Review by dpgiakatos — CHANGES_REQUESTED** [review]
Please follow our code styling for the if statements.
Thank you.

**Comment by dpgiakatos** [comment]
Maybe in the input placeholder is better to have `AS or Prefix` instead of `Search`. What do you think?

File: src/components/charts/NetworkTopologyChart.vue [hunk 1]
```diff
@@ -148,7 +148,7 @@ const init = () => {
 }
 
 const isPrefix = (input) => {
-  if (input.includes('.')) {
+  if (input.includes('.') || input.includes(':')) {
```
Comment (dpgiakatos) [inline src/components/charts/NetworkTopologyChart.vue:151]: Both IPv4 and IPv6 prefixes contain a '/', so you do not need to include conditions for '.' and ':'.

File: src/components/charts/NetworkTopologyChart.vue [hunk 2]
```diff
@@ -430,6 +430,8 @@ watch(
 
 watch(searchInput, (newValue) => {
   emit('searchChange', props.id, newValue)
+  if(newValue.includes('.')) ipModel.value = "IPv4"
```
Comment (dpgiakatos) [inline src/components/charts/NetworkTopologyChart.vue:433]: Here is better to have one if (isPrefix) and then the other two ifs.

**Comment by 0xAnon0602** [comment]
> Maybe in the input placeholder is better to have `AS or Prefix` instead of `Search`. What do you think?

Yeah that's actually a good idea to let user know what exactly to input

**Comment by 0xAnon0602** [comment]
@dpgiakatos Did all the changes you asked for in the new PR )

**Review by dpgiakatos — APPROVED** [review]
Thanks

**Pull request merged**